pkg/gui/controllers/files_controller.go

```diff
@@ -178,10 +180,12 @@ func (self *FilesController) GetKeybindings(opts types.KeybindingsOpts) []*types
 			Description:       self.c.Tr.OpenDiffTool,
 		},
 		{
-			Key:         opts.GetKey(opts.Config.Files.OpenMergeTool),
-			Handler:     self.c.Helpers().WorkingTree.OpenMergeTool,
-			Description: self.c.Tr.OpenMergeTool,
-			Tooltip:     self.c.Tr.OpenMergeToolTooltip,
+			Key:             opts.GetKey(opts.Config.Files.OpenMergeOptions),
+			Handler:         self.withItems(self.createMergeConflictMenu),
```

Comment:
Instead of implementing the menu here, we should implement it in `WorkingTreeHelper.OpenMergeTool` (and rename it accordingly). That way, it will also be used when you are in the merge conflict editor.

Edit: hm, when you are in the merge conflict editor you probably expect to work only on the current file you are in. So we'd have to pass a list of file paths to that function, and pass the selected ones from here, and the current one from merge_conflicts_controller.